From b05bec0f4dd817b1b8579045c457d964a598f6c3 Mon Sep 17 00:00:00 2001
From: Don Dutile <ddutile@redhat.com>
Date: Thu, 9 Nov 2017 06:48:00 -0500
Subject: [PATCH 3/3] [netdrv] qed: RoCE doesn't need to use SRC

Message-id: <20171109065423.8574-86-ddutile@redhat.com>
Patchwork-id: 197064
O-Subject: [PATCH RHEL-7.5 v2 085/468] qed: RoCE doesn't need to use SRC
Bugzilla: 1462433 1499362
RH-Acked-by: Jerry Snitselaar <jsnitsel@redhat.com>
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

commit 5f8cb033f4c814760c9f7f2345278111d06afe47
Author: Mintz, Yuval <Yuval.Mintz@cavium.com>
Date:   Mon Apr 3 12:21:12 2017 +0300

    qed: RoCE doesn't need to use SRC

    As RoCE doesn't need to use the SRC, allocating ILT memory
    on behalf of RoCE is wasting available ILT lines.

    Signed-off-by: Yuval Mintz <Yuval.Mintz@cavium.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1462433

Signed-off-by: Donald Dutile <ddutile@redhat.com>
Signed-off-by: Rafael Aquini <aquini@redhat.com>
---
 drivers/net/ethernet/qlogic/qed/qed_cxt.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

Index: src/drivers/net/ethernet/qlogic/qed/qed_cxt.c
===================================================================
--- src.orig/drivers/net/ethernet/qlogic/qed/qed_cxt.c	2017-12-04 17:13:58.826170322 +0100
+++ src/drivers/net/ethernet/qlogic/qed/qed_cxt.c	2017-12-04 17:14:01.113139043 +0100
@@ -241,8 +241,7 @@
 static bool src_proto(enum protocol_type type)
 {
 	return type == PROTOCOLID_ISCSI ||
-	       type == PROTOCOLID_FCOE ||
-	       type == PROTOCOLID_ROCE;
+	       type == PROTOCOLID_FCOE;
 }
 
 static bool tm_cid_proto(enum protocol_type type)
